Jenkinsfile

```diff
@@ -74,13 +75,17 @@ def job = {
 
         // For a release build we remove the -SNAPSHOT from the version.
         config.ksql_db_version = config.ksql_db_version.tokenize("-")[0]
+
+        config.ksql_db_artifact_version = config.ksql_db_version
         config.docker_tag = config.ksql_db_version
     } else {
-        config.docker_tag = config.ksql_db_version.tokenize("-")[0] + '-' + env.BUILD_NUMBER
+        // For non-release builds, we append the build number to the maven artifacts and docker image tag
+        config.ksql_db_artifact_version = config.ksql_db_version.tokenize("-")[0] + '-' + env.BUILD_NUMBER
+        config.docker_tag = config.ksql_db_version.tokenize("-")[0] + '-beta' + env.BUILD_NUMBER
```

Comment:
I would prefer we didn't use the beta qualifier in the artifact version.

Reply:
As discussed offline, I've updated the docker tag, maven artifact version, and git tag to all use `-rc` instead of `-beta`.